.azure-pipelines/pr_test_scripts.yaml

```diff
@@ -346,10 +346,14 @@ onboarding_t0:
   - arp/test_stress_arp.py
   - arp/test_unknown_mac.py
   - arp/test_wr_arp.py
+  - everflow/test_everflow_per_interface.py
+  - everflow/test_everflow_testbed.py
 
 onboarding_t1:
   - acl/test_acl.py
   - acl/test_stress_acl.py
+  - everflow/test_everflow_per_interface.py
+  - everflow/test_everflow_testbed.py
```

Comment:
If confirmed that it is passing on KVM testbed, no need to add it here.

Purpose of this unblocking `onboarding-` category is to add everything firstly. The results of this category do not block PR merge. Then work on fixing scripts under this category. Then move fixed scripts to the blocking category.

Reply:
OK, I'll add to t0/t1 job

Reply:
@wangxin currently we can't move to t0/t1 job because skip_traffic_test mark will not take effect because conditional mark could only do longest match

Reply:
As we discussed offline, you can directly add the mark to the scripts or cases before the conditional mark improvement is ready.

Reply:
Fixed in latest comments, use a temporary fixture with same name in everflow module to minimize code change, and would easy to modify in the future